Makefile

```diff
@@ -46,11 +46,11 @@ ldflags = "-X github.com/cosmos/cosmos-sdk/version.Name=axelar \
 	-X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
 	-X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(BUILD_TAGS)" \
 	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
-	-X github.com/CosmWasm/wasmd/x/wasm/types/MaxWasmSize=${MAX_WASM_SIZE} \
 	-X github.com/axelarnetwork/axelar-core/x/axelarnet/exported.NativeAsset=$(DENOM) \
 	-X github.com/axelarnetwork/axelar-core/app.WasmEnabled=$(WASM) \
 	-X github.com/axelarnetwork/axelar-core/app.IBCWasmHooksEnabled=$(IBC_WASM_HOOKS) \
 	-X github.com/axelarnetwork/axelar-core/app.WasmCapabilities=$(WASM_CAPABILITIES) \
+	-X github.com/axelarnetwork/axelar-core/app.MaxWasmSize=${MAX_WASM_SIZE} \
```

Comment:
wasm/types.MaxWasmSize couldn't be overridden due to not being a string type (required by `-X`)